migration/migrating_4_2_4/configuring-replication-repository.adoc

```diff
@@ -10,12 +10,20 @@ You must configure an object storage to use as a replication repository. The Clu
 
 The following storage providers are supported:
 
-* Generic S3 object storage, for example, NooBaa, Minio, Ceph S3
-* xref:../../migration/migrating_4_2_4/configuring-replication-repository.adoc#migration-configuring-aws-s3_{context}[AWS S3]
-* xref:../../migration/migrating_4_2_4/configuring-replication-repository.adoc#migration-configuring-gcp_{context}[Google Cloud Provider]
-* xref:../../migration/migrating_4_2_4/configuring-replication-repository.adoc#migration-configuring-azure_{context}[Microsoft Azure]
+* Generic S3 object storage, for example, Minio or Ceph S3
+* Multi-Cloud Object Gateway
```

Comment:
It is better to notice that MCG is the abbreviation of `Multi-Cloud Object Gateway`.